docs/quick-start.rst

```diff
@@ -120,7 +120,7 @@ Now you can execute the benchmark as follows:
 
 .. code-block:: bash
 
-   $ vendor/bin/phpbench run tests/Benchmark/TimeConsumerBench.php --report=default
+   $ php vendor/bin/phpbench run tests/Benchmark/TimeConsumerBench.php --report=default
```

Comment:
why the `php` prefix? `bin/phpbench` already has the shebang?

Reply:
Just because the other commands in this quick start have it 🤔 

If you are fine, I would remove the `php` from `php phpbench ...` completely through the whole docs

Reply:
Yes I think that's best if there is no advantage to it

Reply:
The original document was written many years ago...

Reply:
Can you merge this? I will provide another PR afterwards